README.rst

```diff
@@ -55,26 +57,46 @@ python-udsoncan
 Link: https://github.com/pylessard/python-udsoncan
 
 - pros:
-    - comprehensive documentation
-    - handlers for multiple diagnostic services is implemented
+
+  - comprehensive documentation -
+    https://udsoncan.readthedocs.io/en/latest/index.html
+  - maintained with active community - https://udsoncan.readthedocs.io/en/latest/udsoncan/questions_answers.html
+  - various connection types are supported -
+    https://udsoncan.readthedocs.io/en/latest/udsoncan/connection.html#available-connections
+  - CAN bus fully supported with possibility to extension for other buses (requires custom code)
+  - possibility to configure all transmission parameters for CAN using can-isotp package -
+    https://can-isotp.readthedocs.io/en/latest/isotp/implementation.html#
+  - handlers for multiple diagnostic services are implemented -
+    https://udsoncan.readthedocs.io/en/latest/udsoncan/services.html
+  - positive and negatives scenarios are handled - https://udsoncan.readthedocs.io/en/latest/udsoncan/exceptions.html
+
 - cons:
-    - only positive use case scenarios (where communication is in line with UDS standard) are supported
-    - only CAN bus is supported
-    - only Client side communication can be handled
-    - limited communication parameters configuration
+
+  - does not support simulating negatives scenarios on Transport/Network layer, e.g.
+
+    - cannot force Overflow / Wait value of Flow Status (parameter of Flow Control packet)
+    - cannot send CAN packets in wrong order (e.g. incorrect Sequence Numbers order in Consecutive Frames)
+
+  - only Client side communication is implemented - https://udsoncan.readthedocs.io/en/latest/udsoncan/client.html#
+  - no control or measurement of CAN timing parameters (N_As, N_Ar, N_Bs, N_Br, N_Cs, N_Cr)
```

Comment:
These timings are ISOTP values. Not UDS.   My isotp package has control over timing, they're just given meaningful names.

Reply:
updated
please, have a look again